frida_tools/apk.py

```diff
@@ -313,6 +314,9 @@ def append_str(self, add: str) -> int:
             # Insert a UTF-16 NUL
             chunk_data.extend([0, 0])
 
+        # pad to a multiple of 4 bytes
+        chunk_data.extend([0] * (4 - len(chunk_data) % 4))
```

Comment:
```suggestion
        alignment_padding = [0] * (4 - len(chunk_data) % 4)
        chunk_data.extend(alignment_padding)
```

Reply:
Won't this add 4 bytes of padding when `chunk_data` is already a multiple of four?

Reply:
It did, fixed! 😃